source/_integrations/alarmdecoder.markdown

```diff
@@ -34,6 +34,7 @@ alarmdecoder:
     host: 192.168.1.20
     port: 10000
   panel_display: false
+  autobypass: false
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.